### PR TITLE
Update class photos, some intro copy and misc formatting/refactors

### PR DESCRIPTION
**Pull request by RobbieZhuang**
✨ This PR is best reviewed commit by commit ✨
Screen Shot 2023-05-14 at 3 48 03 PM

### CODE REVIEW

File: content/health.html [hunk 1]
```diff
@@ -1,108 +1,148 @@
...
 <div class="vbx4"></div>
 
 <h3>Imposter Syndrome</h3>
 <p class="half">
-  <span class="big">Over 75% of respondents felt imposter syndrome over their university years.</span>
-  Impostor syndrome refers to an individual's internal experience of believing that they are not as competent as others perceive them to be. 
-  The Software Engineering program is very competitive and has one of the highest admission averages in Waterloo. This can place students in a 
-  pressure-to-achieve environment which might have led to students feeling like an imposter.
+  <span class="big"
+    >Over 75% of respondents felt imposter syndrome over their university
```
Comment (wesley-a-leung) [inline content/health.html]: Honestly I find this formatting a bit weird but it works? 🙃

File: content/introduction.html [hunk 2]
```diff
@@ -17,20 +17,24 @@ <h1>About the program</h1>
...
+<div class="vbx2"></div>
 
 <h1>Preface</h1>
 
 <p class="full">
   The findings in this report were derived from the results of a class survey.
-  In total, 66 out of 92 (72%) graduating students, and 7 who were in SE but
-  transferred out, responded.
-  <!-- // todo: robbie check these numbers  -->
+  In total, 66 graduating students responded.
```
Comment (wesley-a-leung) [inline content/introduction.html]: Do we not know how many students graduated/moved out?

File: content/health.html (hunk 1 above)
Comment (RobbieZhuang) [reply content/health.html]: omg good catch - prettier just be wack

File: content/introduction.html (hunk 2 above)
Comment (RobbieZhuang) [reply content/introduction.html]: i have no clue. but just asked se advisor today, let's see what they say!

**Review by wesley-a-leung — APPROVED** [review]

File: content/introduction.html (hunk 2 above)
Comment (RobbieZhuang) [reply content/introduction.html]: 101!

**Pull request merged**